packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -124,33 +125,43 @@ class Checkbox extends StatefulWidget {
   ///
   /// When [value] is null and [tristate] is true, [MaterialState.selected] is
```

Comment:
Do you mean a capital S in tristate? I haven't seen that pattern being used for parameters.

Reply:
Do you mean a capital S in tristate? I guess but I want to avoid making changes outside of adding my theme if that's cool with you. As the PR is already so big.